hudi-client/src/main/java/org/apache/hudi/table/UserDefinedBulkInsertPartitioner.java

```diff
@@ -31,4 +31,6 @@
 public interface UserDefinedBulkInsertPartitioner<T extends HoodieRecordPayload> {
 
   JavaRDD<HoodieRecord<T>> repartitionRecords(JavaRDD<HoodieRecord<T>> records, int outputSparkPartitions);
+
+  boolean arePartitionRecordsSorted();
```

Comment:
please improve javadocs of this interface while you are here :)

Reply:
Yeah, will do.

Reply:
+1

Reply:
javadoc please